src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Main.cs

```diff
@@ -93,7 +96,7 @@ public string GetTranslatedPluginDescription()
             string timeExample = Resources.Microsoft_plugin_timedate_plugin_description_example_time + "::" + DateTime.Now.ToString("T", CultureInfo.CurrentCulture);
             string dayExample = Resources.Microsoft_plugin_timedate_plugin_description_example_day + "::" + DateTime.Now.ToString("d", CultureInfo.CurrentCulture);
             string calendarWeekExample = Resources.Microsoft_plugin_timedate_plugin_description_example_calendarWeek + "::" + DateTime.Now.ToString("d", CultureInfo.CurrentCulture);
-            return string.Format(CultureInfo.CurrentCulture, Resources.Microsoft_plugin_timedate_plugin_description, Resources.Microsoft_plugin_timedate_plugin_description_example_day, dayExample, timeExample, calendarWeekExample);
+            return string.Format(CultureInfo.CurrentCulture, MicrosoftPluginTimedatePluginDescription, Resources.Microsoft_plugin_timedate_plugin_description_example_day, dayExample, timeExample, calendarWeekExample);
```

Comment:
what about Resources.Microsoft_plugin_timedate_plugin_description_example_day ?

Reply:
The 2nd parameter of the string.Format function is CompositeFormat. The CA1863 error says that we need to use private static readonly CompositeFormat x variable. We do not receive this compile error in other parameters.

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1863